backup/backup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Back up an instance and store the data in the local directory.
+#
+# NOTE that to prevent inconsistent data, the MASTODON SERVICE WILL BE STOPPED for the duration of the backup.
+#
+# Backup can run either locally or remote. To run locally, start this script without options.
+#
+# For remote mode, which uses scp/ssh, supply "user@remote_host" and a private key file as first and second argument.
+
+set -euo pipefail
+backup_basedir="/opt/backup"
+
+remote=""
+
+if [ "$#" -gt 0 ] ; then
+    if [ -n "${1:-}" -a -f "${2:-}" ] ; then
```

Comment:
You could probably make sure here that the file is not empty:
```suggestion
    if [ -n "${1:-}" -a -s "${2:-}" ] ; then
```